src/PAModel/MsAppTest.cs

```diff
@@ -179,10 +180,12 @@ public static bool StressTest(string pathToMsApp)
                         return false;
                     }
 
+                    ErrorContainer errorContainer = new ErrorContainer();
```

Comment:
It looks like an `ErrorContainer` is being created just above this (line 163). Is there any reason we can't use that for the Compare call?